sdks/python/apache_beam/testing/analyzers/perf_analysis.py

```diff
@@ -76,14 +75,17 @@ def run_change_point_analysis(params, test_id, big_query_metrics_fetcher):
   if not change_point_index:
     return
 
-  if not is_change_point_in_valid_window(num_runs_in_change_point_window,
-                                         change_point_index):
+  if not is_change_point_in_valid_window(
+      num_runs_in_change_point_window,
+      len(timestamps) - 1 - change_point_index):
```

Comment:
make a helper variable for `len(timestamps) - 1 - change_point_index)` that will explain what this is (I'm confused actually what it is - `is_change_point_in_valid_window` takes a variable called `change_point_index`, but the meaning of that `change_point_index` would be different from the meaning of `change_point_index` in this code, since apparently we need to do some calculations first, hence also confusion that `change_point_index` means a different  concept in two places in the code. please fix.